### PR TITLE
Cleanup namespaces boost

### PR DESCRIPTION
**Pull request by Nicogene**
This PR cleanup the namespace usage of boost asio in some header

### CODE REVIEW

File: src/blocktestcore/connection.cpp [hunk 1]
```diff
@@ -8,6 +8,7 @@
 #include <chrono>
 
 using namespace std::literals::chrono_literals;
+using namespace boost::asio::ip;
```
Comment (traversaro) [inline src/blocktestcore/connection.cpp:11]: I tought the idea of the PR is to remove the `using namespace`, is this addition intentional?

Comment (traversaro) [reply src/blocktestcore/connection.cpp:11]: Ahh I got it, the idea was to remove it from the headers, make sense.

**Review by traversaro — APPROVED** [review]
Even if it does not fix the problem, it make sense to do this change.

File: src/blocktestcore/connection.cpp (hunk 1 above)
Comment (Nicogene) [reply src/blocktestcore/connection.cpp:11]: I moved it in the cpp that should be harmless in theory, but in case I can get rid of it also from the cpp
